CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v0.23 [2024-01-23]
+
+_What's new?_
+
+- Added the following controls across the benchmarks. ([#88](https://github.com/turbot/steampipe-mod-terraform-aws-compliance/pull/88) [#102](https://github.com/turbot/steampipe-mod-terraform-aws-compliance/pull/102))
+  - `vpc_security_group_restrict_ingress_rdp_all`
+  - `vpc_security_group_restrict_ingress_ssh_all`
+  - `ses_configuration_set_tls_enforced`
```

Comment:
@khushboo9024 could you please sort the controls? Also, for which benchmark were the above controls added?